README.md

```diff
@@ -93,6 +93,23 @@ oneview_ethernet_network 'Eth1' do
 end
 ```
 
+### oneview_fabric
+
+Fabric resource for HPE OneView.
+
+Support only in API300 onwards with variant Synergy.
+
+Action defaults to `:none`, also performs updates on the reserved vlan range.
+
+```Ruby
+oneview_fabric 'Fabric1' do
+  client <my_client>
+  data <resource_data>
+  reserved_vlan_range <vlan_options> # Hash: Usually the 'start' and 'length' of the range
+  action [:none, :set_reserved_vlan_range]
```

Comment:
After you remove the 2 references to the `:none` action in the README, I'm good with this :)